### PR TITLE
Add DescriptionList documentation

### PR DESCRIPTION
**Pull request by follan**
#153

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-154.gearboxjs.preview.cogniteapp.com

File: src/components/DescriptionList/stories/basic.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  SOURCE_TABLE: 'wmate_dba.wmt_location',
+  WMT_LOCATION_ID: '1004',
+  WMT_LOCATION_WORKSTART: '1999-09-01 07:00:00',
+  latestUpdateTimeSource: '1552471210000',
+}
+
+function ExampleComponent(props) {
+  return (
+    <DescriptionList 
+      valueSet=metadata
```
Comment (mrtraser) [inline src/components/DescriptionList/stories/basic.md]: could we have curly bracers here:
```typescript
valueSet={metadata}
```

the same for other stories

Comment (follan) [reply src/components/DescriptionList/stories/basic.md]: Fixed

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-154.gearboxjs.preview.cogniteapp.com

**Pull request merged**